src/page/dataverse/dataverse.tsx

```diff
@@ -0,0 +1,178 @@
+import { useCallback } from 'react'
+import { Button } from '@/component/button/button'
+import { Card } from '@/component/card/card'
+import Tag from '@/component/tag/tag'
+import type { ColorVariant } from '@/component/tag/tag'
+
+import { useTranslation } from 'react-i18next'
+import '../i18n/index'
+import './dataverse.scss'
+
+type DataverseItem = 'dataspace' | 'dataset' | 'service'
+
+type DataverseItemDetails = {
+  type: DataverseItem
+  label: string
+  description: string
+}
+
+const dataverseItems: DataverseItemDetails[] = [
+  {
+    type: 'dataspace',
+    label: 'AG Open Data Space',
+    description: 'Lorem ipsum dolor sit amet, consectetur '
```

Comment:
Note (not explicitly stated in the specification):
- Descriptions can contain any character, including line breaks.
- In addition, I would like to offer the ability to use [Markdown](https://www.markdownguide.org/getting-started/) in descriptions to provide text formatting options, possible image inclusions, as well as the ability to define HTML links.

Reply:
Great. Good choice for [remarkjs/react-markdown](https://github.com/remarkjs/react-markdown). :+1: